assets/sass/components/thank-with-google/_googlesitekit-twg-supporter-wall.scss

```diff
@@ -41,4 +41,11 @@
 			margin-top: 30px;
 		}
 	}
+
+	.googlesitekit-twg-supporter-wall-banner {
+
+		.googlesitekit-twg-supporter-wall-banner__description {
+			display: block;
+		}
```

Comment:
This was added due to the change to a `span` tag correct? I don't think it's needed otherwise (see question above). If it really was needed, it would be better to use the `.googlesitekit-display-block` utility class instead.

Reply:
Sure 👍

Reply:
Updated.